react/components/ShowAvailable.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { useContextCssHandles } from './CssHandlesContext'
+
+interface Props {
+  showAvailabilityMessage?: string
+  availableQuantity: number
+}
+
+function ShowAvailable({ showAvailabilityMessage, availableQuantity }: Props) {
+  const { handles } = useContextCssHandles()
+
+  return (
+    <span className={`${handles.showAvailableText} c-muted-2 t-body b mh1`}>
+      {showAvailabilityMessage} {availableQuantity}
```

Comment:
Also, when you refactor it, don't include this space. Let the users choose if they want to include or not.

```suggestion
      {showAvailabilityMessage}{availableQuantity}
```